parsely/src/main/java/com/parsely/parselyandroid/ParselyTracker.java

```diff
@@ -418,7 +418,7 @@ public void resetVideo() {
     void enqueueEvent(Map<String, Object> event) {
```

Comment:
Minor (🔍): Consider fixing the compilation error on this Javadoc by using `{@link QueueManager#STORAGE_SIZE_LIMIT}` instead of `{@link #STORAGE_SIZE_LIMIT}`.

PS: Having said that, maybe, after we discuss my other comments, this `STORAGE_SIZE_LIMIT` gets removed completely, thus... 🤷